dist/js/page-specific/citysourced-viewer-googleMaps.js

```diff
@@ -1,237 +1,153 @@
-			return filterResults(item, index, 'street_address');
-		},
-
-		filterCountyResults = function (item, index) {
-			return filterResults(item, index, 'administrative_area_level_2');
-		},
-
-		filterResults = function (item, index, query) {
-			var matchArr;
-			if (item.types) {
-				matchArr = $.grep(item.types, function (item, index) {
-					return item === query;
-				});
-			}
-			return matchArr.length ? matchArr : false;
-		},
-
-		/**
-		 * Remove "USA", since this is only for USA addresses.
-		 */
-		removeCountry = function(addressString) {
-			return addressString.replace(', USA', '').replace(', United States', '');
-		},
-
-		/**
-		 * Created the info window for when you click on a map marker.
-		 */
-		getInfoWindowHtml = function(dataItem) {
-			var html = '<div class="maps-info-window">'
-					+ '<h4><a href="/CitySourced/preview/viewer?reportId=' + dataItem.Id + '">' + dataItem.IssueType + '</a></h4>'
-					+ '<ul>'
-					+ '<li>Status: ' + dataItem.StatusType + '</li>'
-					+ '<li>Reported on ' + dataItem.DateCreated + '</li>'
-					+ '</ul>'
-					+ '</div>';
-			return html;
-		},
-
-		/**
-		 * Create the map and autocomplete, and attach up the click and place_changed handler.
-		 */
-		initGoogle = function () {
-			var data = baltimoreCounty.pageSpecific.citySourcedData,
-				nearbyData = baltimoreCounty.pageSpecific.nearbyData,
-				markers = [],
-				mapSettings = {
-					center: {
-						lat: data.Latitude,
-						lng: data.Longitude
-					},
-					scrollwheel: false,
-					zoom: 14,
-					mapTypeId: 'roadmap',
-					mapTypeControl: false,
-					streetViewControl: false
-				},
-				autocompleteSettings = {
-					types: ['geocode']
-				},
-				infoWindow = new google.maps.InfoWindow({
-					content: "holding..."
-				});
-
-			createMap('map', mapSettings);
-
-			var homeMarkerSettings = {
-				Latitude: mapSettings.center.lat, 
-				Longitude: mapSettings.center.lng, 
-				ImageUrl: '/sebin/n/f/icon-marker-my-report.png', 
-				ZIndex: 7999
+	/**
+	 * Adds a marker to the map.
+	 */
+	var createMarker = function createMarker(settings) {
+		var markerSettings = {
+			map: window.map,
+			position: {
+				lat: settings.Latitude * 1,
+				lng: settings.Longitude * 1
+			},
+			animation: google.maps.Animation.DROP
+		};
+
+		if (settings.ImageUrl) {
+			markerSettings.icon = settings.ImageUrl;
+		}
+
+		if (settings.ZIndex) {
+			markerSettings.zIndex = settings.ZIndex;
+			markerSettings.optimized = false;
+		}
+
+		marker = new google.maps.Marker(markerSettings);
+
+		if (settings.InfoWindowHtml) {
+			marker.html = settings.InfoWindowHtml;
+		}
+
+		return marker;
+	};
+
+	/**
+	 * Looks up the street address from the latitude and longitude.
+	 */
+	var reverseGeocode = function reverseGeocode(latitude, longitude, successCallback) {
+		// eslint-disable-next-line global-require
+		require([
+			'esri/tasks/Locator',
+			'esri/geometry/Point'
+		], function locationToAddress(Locator, Point) {
```

Comment:
should we follow the convetion we are using for angular by placing that named fuction below and then passing in the function name?

```javascript
require(['', '', locationToAddress]);

function locationToAddress(Locator, Point) { 
```